CONTRIBUTING.md

```diff
@@ -14,7 +14,8 @@ Part of the GitHub Flow is submitting pull requests. See
 https://help.github.com/articles/using-pull-requests/ for a good overview of
 what pull requests are and how to use them. Internally, we use the _shared
 repository model_ for collaborative development, though we encourage external
-contributors to contribute through the _fork & pull model_.
+contributors to contribute through the _fork & pull model_ and assign a member
+of the core team to review the pull request.
```

Comment:
We just had cohort 1 of the Labs Team submit a round of pull requests to add themselves to https://github.com/hackedu/hackedu/tree/master/meta/labs#lab-directors and all seemed to work out with the current model. I'm going to leave it as-is for now.